openslides/topics/models.py

```diff
@@ -17,19 +18,26 @@ class Topic(RESTModelMixin, models.Model):
     text = models.TextField(blank=True)
     attachments = models.ManyToManyField(Mediafile, blank=True)
 
+    # In theory there could be one then more agenda_item. But support only one.
+    # See the property agenda_item.
+    agenda_items = GenericRelation(Item)
+
     class Meta:
         default_permissions = ()
 
     def __str__(self):
         return self.title
 
+    @classmethod
+    def all_query(cls):
+        return cls.objects.prefetch_related('attachments', 'agenda_items')
+
```

Comment:
Does this refer to *every* model or to every agenda related model (topic, motion, assignment, but not user)? Is it possible subclass the manager only for the three agenda related models?